### DISCUSSION TITLE
[Atum] modale popup media manager enchance

### DISCUSSION THREAD

**Discussion by micker** [joomla/joomla-cms#33620] [2. Cold Features]
I think the media modal manager can enchance a little to use more space and esaier to manage
this is my idea
![image](https://user-images.githubusercontent.com/2539440/117415503-ad823680-af18-11eb-8057-2c9b86ad56ac.png)

i already strat a discussion here, but i didn't find the best way to propose a pr
https://github.com/joomla/joomla-cms/discussions/32039

i try to change 
`modal-body jviewport-height70` for `modal-body jviewport-height80`
and add a scrolling to area

the concept in video
https://www.loom.com/share/e9bd76bb3c8749f384b0bcf515255bba

**Comment by brianteeman** [comment]
or just use the icon and make the images smaller

**Reply by micker** [reply]
yes we can but i didn't tink that realy a complet solution if we compare with actual
https://www.loom.com/share/8a4ad4e10c584d4b8f9a78c4883b7411?focus_title=1&muted=1&from_recorder=1
ex
if we scroll down all folder diseaper and toolbox is fixed not realy cohrent not ?

**Comment by micker** [comment]
do you think its inetresting maybe i try to propose a PR

**Reply by micker** [reply]
@ciar4n what do you this about this ?
